guides/source/layouts_and_rendering.md

```diff
@@ -238,7 +238,7 @@ TIP: This is useful when you're rendering a small snippet of HTML code.
 However, you might want to consider moving it to a template file if the markup
 is complex.
 
-NOTE: This option will escape HTML entities if the string is not HTML safe.
+NOTE: When using `html:` option, HTML entities will be escaped if the string is not marked as HTML safe by using `html_safe` method.  
```

Comment:
`html_safe` is still officially considered private and one people shouldn't call as far as I know.

cc @rafaelfranca